stable/metrics/templates/sqs-exporter.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.sqsExporter.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "metrics.fullname" . }}-sqs
+  labels:
+    {{- include "metrics.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "metrics.name" . }}-sqs
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "metrics.name" . }}-sqs
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9384'
+        {{- with .Values.annotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ template "metrics.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: sqs-exporter
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.sqsExporter.image.repository }}:{{ .Values.sqsExporter.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: metrics
+              containerPort: 9384
+              protocol: TCP
+          env:
+          - name: SQS_QUEUE_NAME_PREFIX
+            value: {{ .Values.sqsExporter.queueNamePrefix }}
+          {{- if .Values.global.sqs.enabled }}
+          - name: SQS_ENDPOINT
+            value: "{{ include "sqsEndpoint" . }}"
+          - name: AWS_ACCESS_KEY_ID
+            value: TBD
```

Comment:
Let's try to be consistent in the example values. Define some and let's re-use the same.
Not need to change all occurrences now, but when we review other files with same vars, let's put same dummy values.